unpub_auth/pubspec.lock

```diff
@@ -0,0 +1,405 @@
+# Generated by pub
```

Comment:
[We don't need to check-in .lock file in case of published packages](https://dart.dev/guides/libraries/private-files#the-rules:~:text=pubspec.lock%20%20%23%20Except%20for%20application%20packages)